_sass/settings/_z-index.scss

```diff
@@ -0,0 +1,12 @@
+$z-index-1:  100;
```

Comment:
whoa! i love this! this would also be great if you this was a team project and then you know there are already set z-index values so teammates aren't creating random z-indexs at every turn.

Reply:
this was new to me too and I thought it was really cool 👍